scripts/install.sh

```diff
@@ -296,6 +298,7 @@ fi
 if [[ $namespace_scope_mode == "true" ]]; then
   info "Enabling namespace scope mode"
   kubectl set env deploy/modelmesh-controller NAMESPACE_SCOPE=true
+  sed -i 's/#- bases\/serving.kserve.io_clusterservingruntimes.yaml/- bases\/serving.kserve.io_clusterservingruntimes.yaml/g' crd/kustomization.yaml
```

Comment:
I don't think this is needed here, it's already done above?

Reply:
This is to reset crd/kustomization.yaml back to original in case the user wants to delete the namespaced installation and deploy a cluster scoped installation using the same set of files.

Reply:
Ah ok sorry I didn't notice that this was reverting.